### PR TITLE
feat: update sentry to 20.7.1

### LINKED ISSUE #115: Sentry v20.6.0
#### Description

Since [sentry version v20.6.0](https://github.com/getsentry/sentry/releases/tag/20.6.0) has been released, it would be nice to be
able to install v20.6.0 also on the cluster.

### PR DESCRIPTION
**Pull request by Mokto**
Close #115

### CODE REVIEW

**Comment by Mokto** [comment]
@J0sh0nat0r when updating I'm having this issue :
"AssertionError: All storage sets must be assigned to a cluster" on all snuba consumers.

Have you also seen this ?

Thanks

File: sentry/values.yaml [hunk 1]
```diff
@@ -8,12 +8,12 @@ user:
...
   sentry:
     repository: getsentry/sentry
-    tag: cc9f7d1
+    tag: 20.7.1
     pullPolicy: IfNotPresent
     # imagePullSecrets: []
   snuba:
     repository: getsentry/snuba
-    tag: 882be95ba0d462a29759a49b2e9aad0c1ce111a9
+    tag: e944bc414bfc97c9836febbfb2f310a3237f91cf
```
Comment (yoan-adfinis) [inline sentry/values.yaml:16]: `20.7.1` is also present. https://hub.docker.com/layers/getsentry/snuba/20.7.1/images/sha256-069d06ffa5ccb85324291e96de822e072663c55a26e5eb6f60164e3b8ae7d34e?context=explore

Comment (Mokto) [reply sentry/values.yaml:16]: I've tried this one and there seems to be an issue there.
The docker image is not accepting to run `api` through `./docker-entrypoint.sh api`

Comment (yoan-adfinis) [reply sentry/values.yaml:16]: I've seen those error messages as well.

Comment (Mokto) [reply sentry/values.yaml:16]: My errors (`AssertionError: All storage sets must be assigned to a cluster`) were also on that docker tag, btw

**Comment by J0sh0nat0r** [comment]
@Mokto Sorry for the slow reply (currently on holiday :P), to fix this we need to update the storage sets list in the snuba settings config map to match the one in the version of Snuba we're targeting (currently my fork should be up-to-date). It would be nice to improve this workflow and I'll look into a solution soon :)

**Comment by Mokto** [comment]
Thanks @J0sh0nat0r ! Enjoy your vacation.

**Pull request merged**